### ISSUE TITLE
Tutorial misleading

### ISSUE THREAD

**Issue by AMDphreak** [angular/angular#33214]
# 📚 Docs or angular.io bug report

### Description

Tutorial says "An Angular application comprises a tree of components, in which each Angular component has a specific purpose and responsibility."

This is not how you use the word "comprises". An Angular application **_is comprised of_** a tree of components. "comprises a tree of components" would mean "An angular app populates a tree of components". The Angular app does not do this. The programmer does this. The application is acted upon; it does not act upon.

The use of "comprises" in the wrong way is legal only because many have made this mistake, and that phrasing yields [problems](https://en.wiktionary.org/wiki/comprise#English).

tl;dr Standard English does not condone the use of passive voice "comprise". There is a standard way to use that term and it should be followed.

### What's the affected URL?**
 https://angular.io/start

**Comment by MrJithil** [comment]
@AMDphreak Do you have suggestion to replace this word?
**An Angular application is consist with a tree of components.**

**Comment by gkalpak** [comment]
@AMDphreak, according to the link you posted `comprise` means _"to be made up of"_, _"to consist of"_. So, it sounds right to me to say that an Angular application _is made up of_/_consists of_ a tree of components 😕

I could agree that the word "comprise" is not a common one and many non-native speakers would have to look it up (and therefore using simpler language in the docs would preferable), but do think it is correct.

<sub>Disclaimer: I am a non-native speaker myself.</sub>

**Comment by MrJithil** [comment]
Hi @gkalpak ,

Please let me know in case we need to change that field for avoid the grammar conflicts

**Comment by gkalpak** [comment]
I'll let someone from the docs team to make the final decision :smiley:

**Comment by julianomacielferreira** [comment]
@gkalpak ,

I think @kapunahelewong is the person in charge to make the decision.

Regards,
Juliano.

**Comment by AMDphreak** [comment]
> @AMDphreak, according to the link you posted `comprise` means _"to be made up of"_, _"to consist of"_. So, it sounds right to me to say that an Angular application _is made up of_/_consists of_ a tree of components 😕
> 
> I could agree that the word "comprise" is not a common one and many non-native speakers would have to look it up (and therefore using simpler language in the docs would preferable), but do think it is correct.
> 
> Disclaimer: I am a non-native speaker myself.

I understand your POV, but can't say I've been there. I am a native English speaker. Nobody uses "comprises" in the way the article does--not in every day language, nor in research articles, nor legalese.

In 13th century England it was correct, but it is not accepted today.

**Comment by kapunahelewong** [comment]
Thanks for raising this point, @AMDphreak. There are two schools of thought on "comprises"/"comprised of" - and [people can be passionate about their choice on it](https://medium.com/backchannel/meet-the-ultimate-wikignome-10508842caad). I've come across "comprises" meaning "made up of" in college and graduate school and on the docs team we prefer it over "comprised of", however, clarity and inclusiveness take precedence, so we can and should change it.

For anyone curious, here are a couple of links that helped inform the choice to use "comprises" over "comprised of":
https://www.merriam-webster.com/words-at-play/can-you-use-comprised-of-grammar
https://www.merriam-webster.com/dictionary/comprise

All that said though, @AMDphreak, your issue inspired me to look it up in the Google writing guidelines and indeed, they advise against using "comprise" at all (though without explanation, and in light of this discussion it really doesn't need it) and in its place recommend "to consist of", "contain", or "include". I would add "made up of", too.

So, @AMDphreak, would you be interested in submitting a PR to change it? I'd be happy to help you get it through the process.

**Comment by AMDphreak** [comment]
> Thanks for raising this point, @AMDphreak. There are two schools of thought on "comprises"/"comprised of" - and [people can be passionate about their choice on it](https://medium.com/backchannel/meet-the-ultimate-wikignome-10508842caad). I've come across "comprises" meaning "made up of" in college and graduate school and on the docs team we prefer it over "comprised of", however, clarity and inclusiveness take precedence, so we can and should change it.

Clarity, I like.

> For anyone curious, here are a couple of links that helped inform the choice to use "comprises" over "comprised of":
> https://www.merriam-webster.com/words-at-play/can-you-use-comprised-of-grammar
> https://www.merriam-webster.com/dictionary/comprise
> 
> All that said though, @AMDphreak, your issue inspired me to look it up in the Google writing guidelines and indeed, they advise against using "comprise" at all (though without explanation, and in light of this discussion it really doesn't need it) and in its place recommend "to consist of", "contain", or "include". I would add "made up of", too.

I didn't know they had a style guideline. That actually is smart of them.

> So, @AMDphreak, would you be interested in submitting a PR to change it? I'd be happy to help you get it through the process.

I'll do that right now.

> EDIT: I may or may not have been in grad school in the 13th century.

Lol

**Comment by kapunahelewong** [comment]
Oh, @AMDphreak, you might find the guidelines interesting then. They're pretty good, imo, though some of my preferences are different than theirs. Still though, they put a lot of thought and work into it. Here's the word list, but the whole site is nice:

https://developers.google.com/style/word-list

**Comment by zarend** [comment]
Thanks for reporting this issue. Luckily it has already been fixed.
